src/controller.js

```diff
@@ -28,6 +31,22 @@ export class Controller {
       this.#view.renderResults(data);
     });
 
+    this.#eventsManager.subscribe('storeGameData', (data) => {
+      this.#storage.setItem(this.#gameDataStorageKey, data);
+    });
+
+    this.#eventsManager.subscribe('getGameData', () =>
+      this.#storage.getItem(this.#gameDataStorageKey)
+    );
+
+    this.#eventsManager.subscribe('gameDataLoaded', (data) =>
+      this.#model.loadScores(data)
+    );
+
+    this.#storage = new Storage((data) =>
+      this.#eventsManager.publish('gameDataLoaded', data)
+    );
```

Comment:
A good break from this was definitely the right idea (or maybe it was the Malbec 😂). It seems while we allowed `Storage` to publish an event when game data is loaded, we forgot to allow it to pass along the game data.

I hope this unblocks work on the other features now, please let me know if anything still breaks unexpectedly!

Reply:
It works now 😂 Thank you for figuring it out. A glass of wine definitely helped! ❤️